src/sentry/grouping/strategies/newstyle.py

```diff
@@ -557,17 +537,6 @@ def single_exception(exception, context, **meta):
                 ],
             )
 
-        if context["detect_synthetic_exception_types"] and _synthetic_exception_type_re.match(
```

Comment:
`detect_synthetic_exception_types` -- this setting is now unused (grep sourcecode for it) -- ideally we'd still use it in event_manager somehow